frontend/utils/trapFocus.js

```diff
@@ -7,8 +7,12 @@ const trapFocus = {
     }
   },
   methods: {
-    focusOnFirst (el = this.$el) {
+    focusEl (el = this.$el) {
+      // prepare the element to be focused programmatically
       Array.from(el.querySelectorAll(this.focusableElements))[0].focus()
```

Comment:
This line can be removed. We do not need to focus the first focusable element, only the el itself.